Gemfile

```diff
@@ -0,0 +1,18 @@
+source 'https://rubygems.org'
+
+gem 'rails', '4.2.5'
+gem 'sqlite3'
+gem 'rspec-rails'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'jbuilder', '~> 2.0'
+gem 'sdoc', '~> 0.4.0', group: :doc
+
+group :development, :test do
+  gem 'byebug'
```

Comment:
let's add 'pry' in here too...we teach students about it earlier on in the curriculum, and most of them rely on it heavily